cdap-docs/tracker-manual/source/index.rst

```diff
@@ -346,12 +416,16 @@ Details on completing this form are described in CDAP's documentation on
 Tracker HTTP RESTful API
 ========================
 
-Tracker supports searching of the *_auditLog* dataset through an HTTP RESTful API. (See
-the :ref:`Reference Manual: HTTP RESTful API <http-restful-api-introduction>` for details
-on the conventions used for this API.) To search for audit log entries for a particular
-dataset, stream, or stream view, submit an HTTP GET request::
+All Tracker features are also available via RESTful endpoints. It supports searching of
+the *_auditLog* dataset, managing preferred tags, and querying metrics, all through a set
+of HTTP RESTful APIs. (See the :ref:`Reference Manual: HTTP RESTful API
```

Comment:
the *_auditLog* dataset, managing preferred tags, and querying metrics, through a set
of HTTP RESTful APIs.